README.md

```diff
@@ -5,17 +5,44 @@
 ### Set Up
 
 1. Visit [chrome://extensions/](chrome://extensions/) and toggle developper mode ON
-2. Click on ```Load unpacked``` and select the WikiExtension folder
+2. Click on `Load unpacked` and select the WikiExtension folder
 
 ### How to do highlighting
 
-Highlighting needs to be given raw content (text, including bolding and links). See the example from the Carleton page. 
+Highlighting needs to be given raw content (text, including bolding and links). See the example from the Carleton page.
 
 ```javascript
-const persistent = ['ranked #1 in Undergraduate Teaching by U.S. News & World Report for over a decade', 'Founded in 1866', 
-                    'Admissions is highly selective', 'Carleton is one of the highest sources of undergraduate students pursuing doctorates'];
+const persistent = [
+    "ranked #1 in Undergraduate Teaching by U.S. News & World Report for over a decade",
+    "Founded in 1866",
+    "Admissions is highly selective",
+    "Carleton is one of the highest sources of undergraduate students pursuing doctorates",
+];
 var arrayLength = persistent.length;
 for (var i = 0; i < arrayLength; i++) {
-    highlightPersistentContent(persistent[i], '#99FF84');
+    highlightPersistentContent(persistent[i], "#99FF84");
 }
-```
\ No newline at end of file
+```
+
+### How to get page count
+
+Use enums.js as inputs for some of the parameters if needed.
+
+```javascript
+import { WIKI_CREATION_DATE, AggregateType } from "./enums.js";
+import getPageViews from "./timeseriesService.js";
+
+// Get page views for the Pasta article since its creation until the beginning of 2022 by day
+const pastaResponse = await getPageViews("Pasta", WIKI_CREATION_DATE, new Date("2022-01-01"), AggregateType.DAILY);
+/*
+    Returns:
+    [
+        [ 2015-07-01T05:00:00.000Z, 2406 ],
```

Comment:
I guess it depends to the input for the graph, but maybe we need to consider returning the date in another format. Eg: 2015-07-01 instead of 2015-07-01T05:00:00.000Z

Reply:
Yup agreed. This is to put everything in a `Date` object, which is the most stable (can be formatted into anything).